python/tvm/relay/op/nn/_nn.py

```diff
@@ -1160,21 +1186,46 @@ def batch_flatten_shape_func(attrs, inputs, _):
 
 
 @script
-def _dense_shape_func(data_shape, weight_shape):
+def _matmul_shape_func(data_shape, weight_shape, data_transposed, weight_transposed):
```

Comment:
nit: the two inputs of matmul are not necessary to be `data` and `weight`, although it's almost right in DNN. We may consider calling them `transpose_a` and `transpose_b` as CuDNN.

Reply:
Updated all `data_transposed` & `weight_transposed` to `transpose_a` & `transpose_b`. And also renamed all the `data` & `weight` in matmul to `tensor_a` & `tensor_b`. Tensor names in dense remain unchanged.